rezerwacje-front/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Switch, Redirect, Route } from 'react-router-dom';
+import { ThemeProvider } from '@material-ui/styles';
+
+import theme from './theme';
+import './App.scss';
+
+import { Main as MainLayout } from './layouts';
```

Comment:
First you are exporting Standard, then you alias it to Main, only to finally alias it to MainLayout right here